tests/models/test_tpu.py

```diff
@@ -383,3 +383,18 @@ def test_tpu_precision_16_clip_gradients(mock_clip_grad_norm, clip_val, tmpdir):
         mock_clip_grad_norm.assert_called()
     else:
         mock_clip_grad_norm.assert_not_called()
+
+
+@RunIf(tpu=True)
+@pl_multi_process_test
+def test_if_test_works_with_checkpoint_false(tmpdir):
+    """
+    Ensure that model trains properly when
+    `checkpoint_callback` is set to False.
+    """
```

Comment:
nit: this fits into 120 characters

```suggestion
    """Ensure that model trains properly when `checkpoint_callback` is set to False."""
```